web-ui/src/main/resources/catalog/templates/gn-typeahead-match.html

```diff
@@ -1 +1,6 @@
-<a tabindex="-1" bind-html-unsafe="match.label | sanitizeHtmlFilter | typeaheadHighlight:query"></a>
+<a tabindex="-1">
+  <span data-ng-if="match.model.record.resourceType"
+        class="fa gn-icon-{{match.model.record.resourceType[0]}}"
```

Comment:
It would be nice to also have class `.fa-fw` here, in order to better align everything after the icon